### PR TITLE
Enhancement: Instruction tooltip for showing custom field data on post

### PR DESCRIPTION
**Pull request by hasinur1997**

### CODE REVIEW

**Comment by tareq1988** [comment]
@hasinur1997 The conflict needs to be resolved

**Comment by hasinur1997** [comment]
@tareq1988 The conflict has been resolved.

File: admin/form-builder/assets/js/form-builder.js [hunk 1]
```diff
@@ -142,6 +143,38 @@
...
                 Vue.nextTick(function () {
                     var el = $('#form-preview-stage .wpuf-form .field-items').eq(payload.toIndex);
+                    if ('yes' == payload.field.is_meta && state.show_custom_field_tooltip) {
+
+                        var image_one  = wpuf_assets_url + '/images/custom-fields/settings.png';
+                        var image_two  = wpuf_assets_url + '/images/custom-fields/advance.png';
+                        var html       = '<div class="wpuf-custom-field-instruction">';
+                            html      += '<div class="step-one">';
+                            html      += '<p style="font-weight: 400">Navigate through<strong><code>WP-admin > WPUF > Settings > Frontend Posting</code></strong>- there you have to check the checkbox: "Show custom field data in the post content area"</p>';
+                            html      += '<img src="'+ image_one +'" alt="settings">';
```
Comment (tareq1988) [inline admin/form-builder/assets/js/form-builder.js:153]: These texts need to be translated, can't be static.

**Review by tareq1988 — CHANGES_REQUESTED** [review]

File: admin/form-builder/assets/js/form-builder.js (hunk 1 above)
Comment (hasinur1997) [reply admin/form-builder/assets/js/form-builder.js:154]: > These texts need to be translated, can't be static.

This has been updated.

**Pull request merged**